pkg/controller/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -334,6 +334,23 @@ func TestReconcileProvisionOK(t *testing.T) {
 				HasLabel("toolchain.dev.openshift.com/type", "code")
 		})
 
+		t.Run("set_label_for_nstemplateset", func(t *testing.T) {
```

Comment:
you don't need to use underscores in the test name

Reply:
@xcoulon sorry! copied it from the tests below: https://github.com/codeready-toolchain/member-operator/pull/142/files/9a97ba0348d4e9206102f847c600c50e748a2a7b#diff-9b35dfc60ea38cb3e546642f4f9cedb4R583 

i've removed it from all the tests though - hope that's okay.

fixed here: 5cc8815b8c2ff4320c17a69bf55d5e7353b91b3f